proto-lens/src/Data/ProtoLens/Service/Types.hs

```diff
@@ -53,6 +55,7 @@ class ( KnownSymbol (ServiceName s)
   type ServicePackage s :: Symbol
   type ServiceMethods s :: [Symbol]
 
+  packedServiceDescriptor :: Proxy s -> B.ByteString
```

Comment:
I think it would be useful to add an analogue to the `messageDescriptor` function from `proto-lens-protobuf-types`:
http://hackage.haskell.org/package/proto-lens-protobuf-types-0.7.0.0/src/src/Data/ProtoLens/Descriptor.hs

```
serviceDescriptor :: forall s . Service s => DescriptorProto
```

Reply:
Thanks for the review. I've added a commit to implement this.